crypto/src/main/kotlin/net/corda/v5/crypto/CompositeKeyGenerator.kt

```diff
@@ -0,0 +1,15 @@
+package net.corda.v5.crypto
+
+import java.security.PublicKey
+
+interface CompositeKeyGenerator {
```

Comment:
As you have `CompositeKeyBuilder` this interface (CompositeKeyGenerator) can be moved to the runtime-os (to crypto-core)

Reply:
Resovled, via further design iteration.